pkg/syncer/util.go

```diff
@@ -259,3 +259,17 @@ func IsMultiAttachAllowed(pv *v1.PersistentVolume) bool {
 	}
 	return false
 }
+
+// GetVolumeMigrationService is a wrapper method in syncer package to initialize migration service
+func GetVolumeMigrationService(ctx context.Context, metadataSyncer *metadataSyncInformer) (migration.VolumeMigrationService, error) {
```

Comment:
Why do we need a wrapper for `migration.GetVolumeMigrationService`? `migration.GetVolumeMigrationService` takes care of instantiating only one instance of VolumeMigrationService.

Reply:
The methods in `VolumeMigrationService` such as GetVolumeID etc., expects VolumeMigrationService to have initialized before invoking the methods. 
To avoid nil pointer dereferences, the packages which use volumeMigrationService needs to make sure the service is initialized. Because syncer is using volumeMigrationService at multiple places, there is a need to check for volumeMigrationService instantiation. To avoid repetitive checks, i have written a wrapper

Reply:
we can directly call `migration.GetVolumeMigrationService`

Reply:
Done